bblfsh/client.py

```diff
@@ -1,54 +1,57 @@
 import os
-import sys
+import typing as t
 
 import grpc
 
-from bblfsh.aliases import (ParseRequest, ParseResponse, NativeParseRequest, NativeParseResponse,
-        VersionRequest, ProtocolServiceStub, SupportedLanguagesRequest, SupportedLanguagesResponse)
-from bblfsh.sdkversion import VERSION
-
-# The following two insertions fix the broken pb import paths
-sys.path.insert(0, os.path.join(os.path.dirname(__file__),
-                                "gopkg/in/bblfsh/sdk/%s/protocol" % VERSION))
-sys.path.insert(0, os.path.dirname(__file__))
+from bblfsh.aliases import (ParseRequest, DriverStub, ProtocolServiceStub,
+                            VersionRequest, SupportedLanguagesRequest, ModeType,
+                            VersionResponse)
+from bblfsh.result_context import ResultContext
 
 
 class NonUTF8ContentException(Exception):
     pass
 
 
-class BblfshClient(object):
+class BblfshClient:
     """
-    Babelfish gRPC client. Currently it is only capable of fetching UASTs.
+    Babelfish gRPC client.
     """
 
-    def __init__(self, endpoint: str):
+    def __init__(self, endpoint:str) -> None:
```

Comment:
This is unrelated to the PR but here is another API request that we really need:

```python
@staticmethod
def wrap(channel, stub_v1, stub_v2) ->  "BblfshClient":
    client = BblfshClient.__new__()
    client._channel = channel
    client._stub_v1 = stub_v1
    client._stub_v2 = stub_v2

def __init__(self, endpoint: str):
    channel = grpc.insecure_channel(endpoint)
    stub_v1 = ProtocolServiceStub(channel)
    stub_v2 = DriverStub(channel)
    return BblfshClient.wrap(channel, stub_v1, stub_v2)
```

Reply:
Can you explain what's the use case of this?

Reply:
Done, thanks!